_sass/_syntax-highlighting.scss

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Do you need all of this stuff? You can poke into Chrome dev tools and check what styles aren't used: http://www.pontikis.net/blog/remove-unused-css